### PR TITLE
INT-12: Front page

### PR DESCRIPTION
**Pull request by sarahsonder**
## Front page

[TASK: INT-12](https://linear.app/uoftblueprint/issue/INT-12/front-page)  

Description of feature or bug:
Front page!

### Important Notes

Add any important notes or commentary necessary for this PR: We're still working on the responsive of the page, but the layout is complete!

### CODE REVIEW

**Comment by vai9er** [comment]
Hi Sarah, looking great so far! Just some notes:

Make sure you are using the proper text color and add some space between the text and the art beneath it!
image

Make the header ("How We Do It") bigger, with respect to the figma prototype:
image

- Make sure to use the footer component on your page!!

- Lastly, put some space between your project card and the blueprint footer. Also make the Current Projects header bigger :)
image

**Comment by vai9er** [comment]
Hi just some more comments:

Please ensure you guys are using Poppins font for your components and text!!

File: public/period-purse2.png
Comment (elsiezhu) [inline public/period-purse2.png:1]: you can do this with
```
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;
```

File: src/app/home-page-top-section/TopSection.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Container, Title, Text, Image } from "@mantine/core";
+import styles from "./TopSection.module.css";
+
+export default function TopSection() {
+  // Top Section of Page
+  // Figure out how to center on home page
+  {
+    console.log(Text.classes);
```
Comment (elsiezhu) [inline src/app/home-page-top-section/TopSection.js:6]: Please remove before merging!

**Review by elsiezhu — COMMENTED** [review]
Hey Sarah and Michelle, the page looks really great! I just had a couple of comments about styling:

- Would it be possible to adjust the width or font size of the Student / Non Profit Partners  sections so that they are aligned?
Screenshot 2024-04-02 at 12 38 12 AM

- Could you also decrease the font size of the card header and description in the current projects section and increase the gap between cards?

File: public/period-purse2.png
Comment (sarahsonder) [reply public/period-purse2.png:1]: Ok! I've made changes to the font size of the Student / Non Profit Partners sections and I've updated the styling for the projects card. Just a quick question - what did you mean by the code above (i.e. `border-bottom-right-radius`)

Comment (elsiezhu) [reply public/period-purse2.png:1]: Oh haha I just noticed that this photo that was edited to match the round edged top / square edged bottom from figma, no biggie though!

**Review by elsiezhu — APPROVED** [review]
Looks good to me!!

File: public/period-purse2.png
Comment (sarahsonder) [reply public/period-purse2.png:1]: Ah okok :)

**Pull request merged**